tests/Oryx.BuildImage.Tests/DotNetCoreSampleAppsTest.cs

```diff
@@ -472,6 +472,49 @@ public void Build_CopiesContentCreatedByPreAndPostBuildScript_ToImplicitOutputDi
                 result.GetDebugInfo());
         }
 
+        [Fact]
+        public void Build_Executes_InlinePreAndPostBuildCommands()
```

Comment:
I don't see a lot of value from having the same test run on two different platforms - the code is shared among all platforms anyway, isn't it?

Reply:
The thing is DotNetCore is no longer using the common base bash script as other platforms. This is because the difference in how .net core deployment works (yeah I don't like it, but until we find a solution which works for all platforms, we need to have a test to verify there too)

Reply:
Understood 👍